engine/chain/internal/bhs/get_merkleroots_from_bhs.go

```diff
@@ -0,0 +1,64 @@
+package bhs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"syscall"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/models"
+)
+
+// GetMerkleRootsFromBHS returns Merkle Roots from Block Header Service
+func (s *Service) GetMerkleRootsFromBHS(ctx context.Context, query url.Values) (*models.MerkleRootsBHSResponse, error) {
+	bhsURL, err := s.createBHSURL("/chain/merkleroot")
```

Comment:
You could adjust also `makeVerifyMerkleRootsRequest` to use the `createBHSURL` helper method.

Reply:
done